replicate-to-sql-with-generic-events/scripts/count-data.js

```diff
@@ -0,0 +1,27 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+const { PostgresWrapper, pgConfigLocal } = require('../lib/postgres');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+async function countFromPostgres() {
+  const postgres = new PostgresWrapper(pgConfigLocal);
+  const pgResponse = await postgres.countData();
+  postgres.disconnect();
```

Comment:
This call should be awaited, if the promise is rejected, it will trigger an unhandledrejection